src/OpenTracing/Standard/Tracer.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace OpenTracing\Standard;
+
+use OpenTracing;
+
+class Tracer implements OpenTracing\Tracer {
```

Comment:
(per my higher-level comment, the "standard" stuff most likely belongs in its own package as a reference impl... see also the extremely early (i.e., hours old) work at https://github.com/opentracing/basictracer-go. Ideally the "basic" impl for PHP would interoperate with the basic impl for Go (etc) at the wire level.